docker-hub/download-rate-limit.md

```diff
@@ -61,23 +61,27 @@ To get a token with a user account (if you are authenticating your pulls) - don'
 $ TOKEN=$(curl --user 'username:password' "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
 ```
 
-Then to get the headers showing your limits, run the following (keep in mind that requesting a manifest emulates a pull and will count against the limits):
+Then to get the headers showing your limits, run the following:
 
 ```
-$ curl -v -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit
+$ curl --head -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit || echo "No RateLimit headers found (pull not limited)"
```

Comment:
One concern here:

```
curl --head -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit
```

If $TOKEN isn't set (or invalid), we're eating any errors.

We are hiding the fact that it returned:

```
HTTP/1.1 401 Unauthorized
```

Reply:
Yeah that's true. I think it's better just to remove it and not try to be fancy, and rely on the next session talking about the lack of headers.

Reply:
This seems to work:

```
curl --head -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep "RateLimit\|HTTP.*[3-9][0-9][0-9]\|resolve" || echo "No RateLimit headers found (pull not limited)"
```

Checking for 300-500 errors (which can happen if $TOKEN is unset/invalid), or for "Could not resolve host"

Reply:
Shawn and I discussed in Slack - better to remove it than give a complicated example that takes into account all possibilities. If people don't see headers they will hopefully read the next section a couple lines down.